Library/Formula/rapidjson.rb

```diff
@@ -0,0 +1,15 @@
+class Rapidjson < Formula
+  homepage "https://miloyip.github.io/rapidjson/"
+  url "https://github.com/miloyip/rapidjson/archive/v1.0.0.tar.gz"
+  sha256 "4189b32b9c285f34b37ffe4c0fd5627c1e59c2444daacffe5a96fdfbf08d139b"
+  head "https://github.com/miloyip/rapidjson.git"
+
+  def install
+    prefix.install "include", "example", "doc"
```

Comment:
Sorry. When I wrote the original formulae there was no cmake support in rapidjson...
I've updated the formulae to use cmake.